examples/edge_intelligence/windmill_graphite_sklearn.py

```diff
@@ -130,145 +130,144 @@ def action_actuator(value):
-
-        rpm_metric = Metric(
-            name="windmill.RPM",
-            unit=None,
-            interval=0,
-            aggregation_size=1,
-            sampling_function=lambda: get_rpm(sensor_tag_collector)
-        )
-        reg_rpm_metric = graphite.register(rpm_metric)
-        graphite.create_relationship(reg_sensor_tag, reg_rpm_metric)
-        reg_rpm_metric.start_collecting()
-
-        pfa_rpm_metric = Metric(
-            name="windmill.RPM",
-            unit=None,
-            interval=0,
-            aggregation_size=1,
-            sampling_function=get_rpm_for_model
-        )
-
-        edge_component = SKLearnEdgeComponent(
-            '/home/pi/Desktop/Borathon/liota/edge_intelligence_models/windmill/finalized_model.sav',
-            actuator_udm=action_actuator)
-
-        pfa_reg_rpm_metric = edge_component.register(pfa_rpm_metric)
-        pfa_reg_rpm_metric.start_collecting()
-
-    except RegistrationFailure:
-        print "Registration to graphite failed"
-        sensor_tag_collector.stop()
+	# create a data center object, graphite in this case, using websocket as a transport layer
+	graphite = Graphite(SocketDccComms(ip=config['GraphiteIP'],
+									   port=config['GraphitePort']))
+
+	try:
+		# create a System object encapsulating the particulars of a IoT System
+		# argument is the name of this IoT System
+		edge_system = Dell5KEdgeSystem(config['EdgeSystemName'])
+
+		# resister the IoT System with the graphite instance
+		# this call creates a representation (a Resource) in graphite for this IoT System with the name given
+		reg_edge_system = graphite.register(edge_system)
+
+		# Operational metrics of EdgeSystem
+		cpu_utilization_metric = Metric(
+			name="windmill.CPU_Utilization",
+			unit=None,
+			interval=10,
+			aggregation_size=2,
+			sampling_function=read_cpu_utilization
+		)
+		reg_cpu_utilization_metric = graphite.register(cpu_utilization_metric)
+		graphite.create_relationship(reg_edge_system, reg_cpu_utilization_metric)
+		# call to start collecting values from the device or system and sending to the data center component
+		reg_cpu_utilization_metric.start_collecting()
+
+		cpu_procs_metric = Metric(
+			name="windmill.CPU_Process",
+			unit=None,
+			interval=6,
+			aggregation_size=8,
+			sampling_function=read_cpu_procs
+		)
+		reg_cpu_procs_metric = graphite.register(cpu_procs_metric)
+		graphite.create_relationship(reg_edge_system, reg_cpu_procs_metric)
+		reg_cpu_procs_metric.start_collecting()
+
+		mem_free_metric = Metric(
+			name="windmill.Memory_Free",
+			unit=None,
+			interval=10,
+			sampling_function=read_mem_free
+		)
+		reg_mem_free_metric = graphite.register(mem_free_metric)
+		graphite.create_relationship(reg_edge_system, reg_mem_free_metric)
+		reg_mem_free_metric.start_collecting()
+		'''
+		# Connects to the SensorTag device over BLE
+		sensor_tag_collector = SensorTagCollector(device_name=config['DeviceName'], device_mac=config['DeviceMac'],
```

Comment:
Chane sensor tag implementation as per the PR

https://github.com/vmware/liota/pull/90